lib/crowbar/init/helpers.rb

```diff
@@ -208,9 +208,8 @@ def api_constraint(*versions)
           break if versions_requested.nil?
           return true if versions_requested[:major].to_i == major &&
               versions_requested[:minor].to_i <= minor
+          halt 406, { "Content-Type" => "application/vnd.crowbar.v#{major}.#{minor}+json" }, ""
```

Comment:
you're right, but the bug was another one anyways. we originally wanted to return the latest servable version in the `Content-Type` header, so returning the requested one (which might not be servable) was just wrong. I restructured the commit with some explanation about it.